CHANGELOG.md

```diff
@@ -1,3 +1,15 @@
+## 🎉 Features 🎉
+- Added ability for a user to allow records having nulls in url fields to be included in the `ad_reporting__url_report` and/or the underlying `...url_report` models. This is done by setting the below variable to `False` in your `dbt_project.yml` file. 
```

Comment:
you can use the wildcard asterisk to denote all url reports:

`*url_report`

Reply:
and also, remember to cite the PR here!

Reply:
done